Examples/JamitFoundationExample/App/Sources/ListView/SampleListViewController.swift

```diff
@@ -0,0 +1,88 @@
+//  Copyright © 2020 Jamit Labs GmbH. All rights reserved.
+
+import JamitFoundation
+import UIKit
+
+final class SampleListViewController: ListViewController<ListItemView> {
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        title = "ListViewController"
+
+        model = ListViewModel.init(
```

Comment:
You can remove `ListViewModel`. `.init(..)` is fine.

Reply:
Will be resolved by the adjustments for the comment above.